text/0006-architecture-review-process.md

```diff
@@ -41,14 +43,20 @@ The Review Packet includes:
     - what review and discussion of the proposed architecture has already happened (and with whom)
   * A competitive analysis suggesting alternatives, costs, and opportunities
 
-3. The chair creates a list of questions to be discussed during the review. These questions should be:
+3. The chair or reviewer creates a list of questions to be discussed during the review. These questions should be:
```

Comment:
Isn't this something that the reviewer should create? After all, it's the reviewer whose questions will need to be addressed. A chair might be able to collate questions from a review panel, but they presumably aren't the subject-matter expert(s) who signed up to generate those questions.